backend/app.js

```diff
@@ -1,10 +1,14 @@
 const express = require('express');
+const dbConnect = require('./dbconnection/dbconnect')
```

Comment:
I would suggest that the database connection be made inside the file rather than creating an external folder just for the purpose.

Thank you for this though